src/components/DatePicker/DatePicker.js

```diff
@@ -47,6 +47,16 @@ export function DatePicker(props) {
     "12",
   ];
 
+  useEffect(() => {
+    // blur the input element on scroll instead of changing the value! Does not affect Keyboard input.
+    document.addEventListener("wheel", function (event) {
+      const el = document.activeElement;
+      if (el?.type === "number") {
+        el.blur();
+      }
+    });
```

Comment:
Can you please add a `return` for a `removeEventListener` call for the element? This will help destroy any bindings to the element that may be left over.